### ISSUE TITLE
Demo site: has colour combinations which have insufficient contrast

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_microsites#507]
On the https://site3.demo.microsites.localgovdrupal.org/ site, the gold and white colour combination is not high enough contrast at small text sizes to meet WCAG Level AA. 

We should update this!
